webapp/src/action_types/index.js

```diff
@@ -17,9 +17,11 @@ export default {
     RECEIVED_GITHUB_USER: pluginId + '_received_github_user',
     RECEIVED_SHOW_RHS_ACTION: pluginId + '_received_rhs_action',
     UPDATE_RHS_STATE: pluginId + '_update_rhs_state',
-    CLOSE_CREATE_ISSUE_MODAL: pluginId + '_close_create_modal',
-    OPEN_CREATE_ISSUE_MODAL: pluginId + '_open_create_modal',
-    OPEN_CREATE_ISSUE_MODAL_WITHOUT_POST: pluginId + '_open_create_modal_without_post',
+    CLOSE_CREATE_OR_UPDATE_ISSUE_MODAL: pluginId + '_close_create_or_update_issue_modal',
+    CLOSE_CLOSE_OR_REOPEN_ISSUE_MODAL: pluginId + '_close_close_or_reopen_issue_modal',
+    OPEN_CREATE_ISSUE_MODAL_WITH_POST: pluginId + '_open_create_issue_modal_with_post',
```

Comment:
do we need a close for  ```OPEN_CREATE_ISSUE_MODAL_WITH_POST```

Reply:
`CLOSE_CREATE_OR_UPDATE_ISSUE_MODAL` is working as close for `OPEN_CREATE_ISSUE_MODAL_WITH_POST`